src/registers.rs

```diff
@@ -44,7 +44,7 @@ impl<'c, 'probe> Registers<'c, 'probe> {
     ) -> anyhow::Result</* cfa_changed: */ bool> {
         match rule {
             CfaRule::RegisterAndOffset { register, offset } => {
-                let cfa = (i64::from(self.get(gimli2probe(register))?) + offset) as u32;
+                let cfa = (i64::from(self.get(gimli2probe(register))?) + offset) as u32; // wrapping_add_signed
```

Comment:
The conversion to i64 and back to u32 could be avoided by just using:
```
   let cfa = self.get(gimli2probe(register))?.wrapping_add_signed(offset);
```
Unfortunately, that's still unstable: https://github.com/rust-lang/rust/issues/87840

When converting this to u64, it will become complicated. Without wrapping_add_signed, the best way probably is some helper function like this: https://github.com/probe-rs/probe-rs/blob/master/probe-rs/src/debug/debug_info.rs#L1414-L1423

Reply:
Thank you. So maybe we can reword the comment to "can be simplified if `wrapping_add_signed` becomes stable"?